src/class-wpml-gutenberg-integration.php

```diff
@@ -298,4 +298,15 @@ public function should_body_be_translated_filter( $translate, WP_Post $post, $co
 	private function is_gutenberg_post( WP_Post $post ) {
 		return (bool) preg_match( '/' . self::GUTENBERG_OPENING_START . '/', $post->post_content );
 	}
+
+	private function parse_blocks( $content ) {
+		global $wp_version;
+		if ( version_compare( $wp_version, '5.0-beta1', '>=' ) ) {
```

Comment:
@BruceOnTheGo why you can't use `version_compare( $wp_version, '5.0', '>=' )`?

Reply:
@andreasciamanna Because beta versions are considered to be less than version without beta.

eg.
`version_compare( '5.0-beta1', '5.0', '>=' );` will return `false`

http://php.net/manual/en/function.version-compare.php

Reply:
Yes, I also checked this `version_compare( '5.0-beta1', '5.0', '>=' )` when put +1

Reply:
For the record, I also checked this and I don't think it needs to be adjusted before the 5.0 release.  
This also works properly when used with the Gutenberg plugin (WordPress 4.9.8 currently)